### DISCUSSION TITLE
connect cloudflare

### DISCUSSION THREAD

**Discussion by mohamedhesham221** [community/community#37108] [Other Feature Feedback, Questions, & Ideas]
why i can't connect my account on cloudflare with github ?
i receive unicorn issue

**Comment by yoannchaudet** [comment]
Hello,

I am not sure what you mean by connecting your account with Cloudflare.

Are you having authentication issues? Or GitHub Pages issues? Don't hesitate to post a screenshot.

**Reply by mohamedhesham221** [reply]
When i try to connect cloudflare pages to my github repos for deploying a web page.
This massege just appears 

![Screenshot_20221025-061358_Firefox.png](https://user-images.githubusercontent.com/67207376/197681186-0aaba943-35cb-4976-b13c-84da9b2581dd.png)

**Reply by mohamedhesham221** [reply]
And i reload page over and over and
Nothing gonna change

**Reply by yoannchaudet** [reply]
That definitely looks like an authentication issue not a Pages one.

I suggest opening a support ticket right here: https://support.github.com/request/account?tags=hubberfy_authentication.

Someone will be able to help you there.
